DESCRIPTION

```diff
@@ -34,7 +34,7 @@ Suggests:
     magrittr,
     png,
     rmarkdown,
-    rtables (>= 0.5.1),
+    rtables (>= 0.6.0.9004),
```

Comment:
why is this in the suggests? @donyunardi

Reply:
I am not too sure. 

Judging from this line:
https://github.com/insightsengineering/teal.widgets/blob/5c1bc697a2c422e6041cbb85bf80614316b83adb/R/table_with_settings.R#L78-L80

and several other lines that is using rtables function in `basic_table_args.R`, it sounds plausible to move rtables from Suggest to Imports.

I'll open an issue so we can discuss and not holding up this PR.

Reply:
I've also seen this approach in other packages. Look at this one - shiny is in Suggests as it is only used in one of of 30 functions. shiny has at least 20 dependencies, so by adding shiny to Imports you would actually be adding 20 packages with all their dependencies, just to support one function. It's a tradeoff that is quite common in R world
https://github.com/pbiecek/archivist/blob/master/DESCRIPTION#L42
https://github.com/pbiecek/archivist/blob/master/R/shinySearchInLocalRepo.R#L57